lib/iex/test/iex/pry_test.exs

```diff
@@ -180,6 +196,7 @@ defmodule IEx.PryTest do
       assert IEx.Pry.breaks() == [{1, URI, {:decode_query, 2}, 1}]
     end
 
+    @tag :require_ast
```

Comment:
Now I am thinking... what if we do the opposite and exclude both IEx.Pry and IEx.Introspection from coverage analysis?

Because if we skip them, it means their coverage will be wrong and we don't get to see the result of the test either. By skipping those modules from cover, then we at least get the tests to actually run.

Reply:
Actually, Both Pry and Introspection should work even with cover compilation, just not if applied to a cover compiled module. In the best case, we would change the tests to use a different subject module.

Same with the `FunctionClauseError` module clauses.

It would be great to have a specific subject test module that can be used.

Reply:
Do we have something similar to `elixirc_paths` available where we can include a module only for tests?

Reply:
You can see what we do in Mix suite: https://github.com/elixir-lang/elixir/blob/3dc3f4e3f16702a04de0e37ffad2b217bc5daa28/lib/mix/test/test_helper.exs#L476

Decoupling these from the actual source files would be positive indeed.

Reply:
That works great. I've adapted the tests that make sense. Now only the `Kernel.DialyzerTest` is completely disabled and a few single test cases.